ui/k-info-panel/switch_info.kytos

```diff
@@ -61,18 +116,102 @@
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+
+     rowClicked (link) {
+       var subtitle = ""
+       if(link.metadata.link_name !== undefined && link.metadata.link_name.length !== 0){
+         subtitle = link.metadata.link_name
+       }
+       else{
+         subtitle = link.id
+         if(subtitle.length > 16) subtitle=subtitle.substr(0,16) + "..."         
+         
+       }
+       var content = {
+                      "component": 'kytos-topology-k-info-panel-link_info',
+                      
+                      "icon": "home",
+                      "title": "Link Details",
+                      "subtitle": subtitle,
+                      "content": link,
+       }
+       this.$kytos.$emit("showInfoPanel", content)
+     },
    },
    mounted () {
      this.update_switch_content()
+     this.get_links()
```

Comment:
There is a bug that if you keep the Link accordion opened, and click another switch, the link list is not rerendered.
I suggest to move this code inside the update_switch_content. I think with this the `watch content` will do the trick.

Reply:
Now it its changed. Commit [848f8dca7f90cbed52993e5edd9ea08e7fa164c3](https://github.com/kytos-ng/topology/pull/73/commits/848f8dca7f90cbed52993e5edd9ea08e7fa164c3)